interface/weno/validate.php

```diff
@@ -16,110 +16,127 @@
+    ++$i;
 }
 
-if (empty($facility['fax'])) {
-    print xlt("Please fill out facility fax properly");
-    exit;
+if (empty($facility[0]['fax'])) {
+    print xlt("Please fill out facility fax properly"."<br>");
+    ++$i;
 }
 
-if (empty($facility['street'])) {
-    print xlt("Please fill out facility street properly");
-    exit;
+if (empty($facility[0]['street'])) {
+    print xlt("Please fill out facility street properly")."<br>";
+    ++$i;
 }
 
-if (empty($facility['city'])) {
-    print xlt("Please fill out facility city properly");
-    exit;
+if (empty($facility[0]['city'])) {
+    print xlt("Please fill out facility city properly")."<br>";
+    ++$i;
 }
 
-if (empty($facility['state'])) {
-    print xlt("Please fill out facility state properly");
-    exit;
+if (empty($facility[0]['state'])) {
+    print xlt("Please fill out facility state properly")."<br>";
+    ++$i;
 }
 
-if (empty($facility['postal_code'])) {
-    print xlt("Please fill out facility postal code properly");
-    exit;
+if (empty($facility[0]['postal_code'])) {
+    print xlt("Please fill out facility postal code properly")."<br>";
+    ++$i;
 }
 
 if (empty($GLOBALS['weno_account_id'])) {
     print xlt("Weno Account ID information missing")."<br>";
-    exit;
+    ++$i;
 }
 if (empty($GLOBALS['weno_provider_id'])) {
     print xlt("Weno Account Clinic ID information missing")."<br>";
-    exit;
+    ++$i;
 }
 if (empty($patient['DOB'])) {
     print xlt("Patient DOB missing"). "<br>";
-    exit;
+    ++$i;
 }
 if (empty($patient['street'])) {
     print xlt("Patient street missing"). "<br>";
-    exit;
+    ++$i;
 }
 if (empty($patient['postal_code'])) {
     print xlt("Patient Zip Code missing"). "<br>";
-    exit;
+    ++$i;
 }
 if (empty($patient['city'])) {
     print xlt("Patient city missing"). "<br>";
-    exit;
+    ++$i;
 }
 if (empty($patient['state'])) {
     print xlt("Patient state missing"). "<br>";
-    exit;
+    ++$i;
 }
 if (empty($patient['sex'])) {
     print xlt("Patient sex missing"). "<br>";
-    exit;
+    ++$i;
 }
-if (empty($pharmacy['name'])) {
-    print xlt("Pharmacy not assigned to the patient"). "<br>";
-    exit;
+if (empty($diagnosis)) {
+    print xlt("Please enter a Medical Problem for this patient"). "<br>";
 }
+
+
 $ncpdpLength = strlen($pharmacy['ncpdp']);
 if (empty($pharmacy['ncpdp']) || $ncpdpLength < 7) {
     print xlt("Pharmacy missing NCPDP ID or less than 7 digits"). "<br>";
-    exit;
+    ++$i;
 }
 $npiLength = strlen($pharmacy['npi']);
 if (empty($pharmacy['npi'] || $npiLength < 10)) {
     print xlt("Pharmacy missing NPI  or less than 10 digits"). "<br>";
-    exit;
+    ++$i;
 }
 //validate NPI exist
 //Test if the NPI is a valid number on file
 $seekvalidation = $validation->validateNPI($pharmacy['npi']);
 if ($seekvalidation == 0) {
     print xlt("Please use valid NPI");
-    exit;
+    ++$i;
+}
+if ($i < 1) {
+    header('Location: prescriptionOrder.php');
```

Comment:
Remove the cap in this name (O is capitalized)

Reply:
removed